Tests/SwiftDocCTests/Model/DocumentationMarkupTests.swift

```diff
@@ -158,6 +158,22 @@ class DocumentationMarkupTests: XCTestCase {
             XCTAssertEqual(expected, model.abstractSection?.content.map({ $0.detachedFromParent.debugDescription() }).joined(separator: "\n"))
         }
 
+        // Contains a HTMLBlock before the abstract
+        do {
+            let source = """
+            # Title
+            <!--Line a-->
+            Line b
+            ## Hello, world!
+            Discussion content.
+            """
+            let explected = """
+            Text \"Line b\"
+            """
+            let model = DocumentationMarkup(markup: Document(parsing: source, options: .parseBlockDirectives))
+            XCTAssertNotNil(model.abstractSection)
+            XCTAssertEqual(explected, model.abstractSection?.content.map({ $0.detachedFromParent.debugDescription() }).joined(separator: "\n"))
```

Comment:
Just a small typo here:

```suggestion
            XCTAssertEqual(expected, model.abstractSection?.content.map({ $0.detachedFromParent.debugDescription() }).joined(separator: "\n"))
```

And we don't need the assertion on 174 anymore right? It's covered by the next line?